sec2_the-ruby-programming-language/2.5_methods.rb

```diff
@@ -0,0 +1,33 @@
+def multiply(first, second)
+    first.to_f * second.to_f    # automatically return the last line, 
+                                # hence no need to explicitly call "return"
```

Comment:
```suggestion
    # automatically return the last line, 
    # hence no need to explicitly call "return"
    first.to_f * second.to_f    
```

Put comments to the top